### PR TITLE
fix: code changes needed for webpack on utils

### PR DESCRIPTION
**Pull request by lcampos**
### What does this PR do?
Updating code references (mainly) before re-introducing the use of webpack on the utils module

### What issues does this PR fix or reference?
@W-8183039@

### CODE REVIEW

File: packages/salesforcedx-utils-vscode/src/cli/cliConfiguration.ts [hunk 1]
```diff
@@ -28,15 +26,6 @@ export function isCLIInstalled(): boolean {
   return isInstalled;
 }
 
-export function showCLINotInstalledMessage() {
```
Comment (lcampos) [inline packages/salesforcedx-utils-vscode/src/cli/cliConfiguration.ts:31]: This code is not being used so I'm removing it for now.

File: packages/salesforcedx-utils-vscode/src/workspaces/index.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/*
```
Comment (lcampos) [inline packages/salesforcedx-utils-vscode/src/workspaces/index.ts:1]: With webpack we make the assumption that a module's entry point will be its index.ts file.

File: packages/salesforcedx-utils-vscode/src/telemetry/telemetryReporter.ts [hunk 3]
```diff
@@ -8,12 +8,12 @@ import * as appInsights from 'applicationinsights';
 import * as fs from 'fs';
 import * as os from 'os';
 import * as path from 'path';
-import * as vscode from 'vscode';
```
Comment (xyc) [inline packages/salesforcedx-utils-vscode/src/telemetry/telemetryReporter.ts:11]: so this doesn't work for webpack? `import * as vscode from 'vscode'`

**Review by xyc — APPROVED** [review]

**Review by AnanyaJha — APPROVED** [review]

File: packages/salesforcedx-utils-vscode/src/telemetry/telemetryReporter.ts (hunk 3 above)
Comment (lcampos) [reply packages/salesforcedx-utils-vscode/src/telemetry/telemetryReporter.ts:11]: This change wasn't specifically for webpack but for finding which things need to be mocked when writing the tests.

**Pull request merged**